packages/app-webdir-ui/src/SearchPage/index.js

```diff
@@ -98,10 +100,16 @@ function SearchPage({
     }
   };
 
+  const componentMap = {
+    FacultyTab: <PreFacStaffMsg />,
+    StudentsTab: <PreStudentMsg />,
+  };
+
   const preSearchOrContent = content => {
     if (term) return content;
 
-    return <PreSearchMessage />;
+    const component = componentMap[content.type.name];
```

Comment:
Weird. I'll look into this more...

Reply:
Thanks @davidornelas11 for looking at this. I have no idea how to fix it. Haha. Maybe I need to hand this ticket off to you or Scott or Mayco. I thought it would be a fairly simple fix, but I guess it's more complicated. Dang. Oh well. It was worth a shot. 😄